### PR TITLE
initial panconvert cask

### PR DESCRIPTION
**Pull request by napaxton**
Getting a error message:

```
$ brew cask install panconvert
==> Downloading http://sourceforge.net/projects/panconvert/files/latest/download
######################################################################## 100.0%
==> No checksum defined for Cask panconvert, skipping verification
Error: It seems the symlink source is not there: '/opt/homebrew-cask/Caskroom/panconvert/latest/PanConvert.app'
```

### CODE REVIEW

File: Casks/panconvert.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  url "http://sourceforge.net/projects/panconvert/files/latest/download?source=files"
+  appcast 'https://github.com/apaeffgen/PanConvert/blob/master/docs/changelog.md',
+      checkpoint: 'e2333a9cd663739863c8306193960401977aaa88'
+
+  name 'PanConvert'
+  homepage 'http://panconvert.sourceforge.net'
+  license :gpl
+
+  app 'PanConvert.app'
+end
```
Comment (vitorgalvao) [inline Casks/panconvert.rb]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

File: Casks/panconvert.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+cask 'panconvert' do
+  version :latest
+  sha256 :no_check
+
+  url "http://sourceforge.net/projects/panconvert/files/latest/download?source=files"
```
Comment (vitorgalvao) [inline Casks/panconvert.rb]: You haven’t changed it like I said. This won’t work.

Comment (napaxton) [inline Casks/panconvert.rb]: Oh, shoot. I did, but then I couldn't commit the changes (I'm not a vim person, and the commit message via vim…), and it must have reverted. Will fix later today.

-N

Comment (vitorgalvao) [inline Casks/panconvert.rb]: `git` uses `$EDITOR`, so whatever you set via `export EDITOR` will be used ([more details](https://unix.stackexchange.com/questions/77089/what-is-the-editor-command-in-bash)).

File: Casks/panconvert.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+cask 'panconvert' do
+  version :latest
+  sha256 :no_check
+
+  url "http://downloads.sourceforge.net/sourceforge/panconvert/PanConvert-0.1.5.dmg"
+  appcast 'https://github.com/apaeffgen/PanConvert/blob/master/docs/changelog.md',
+      checkpoint: 'e2333a9cd663739863c8306193960401977aaa88'
+
```
Comment (vitorgalvao) [inline Casks/panconvert.rb]: This empty line must go.

File: Casks/panconvert.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+cask 'panconvert' do
+  version :latest
+  sha256 :no_check
+
+  url "http://downloads.sourceforge.net/sourceforge/panconvert/PanConvert-0.1.5.dmg"
```
Comment (vitorgalvao) [inline Casks/panconvert.rb]: Do instead `url "http://downloads.sourceforge.net/sourceforge/panconvert/PanConvert-#{version}.dmg"`.

File: Casks/panconvert.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+cask 'panconvert' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/panconvert.rb]: Should be `version '0.1.5'`.

File: Casks/panconvert.rb [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/panconvert.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

**Comment by adidalal** [comment]
When you're done, don't merge, rebase and squash your commits.
https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history or https://davidwalsh.name/squash-commits-git if you need help

File: Casks/panconvert.rb (hunk 1 above)
Comment (napaxton) [inline Casks/panconvert.rb]: so just add "\n" after "end"?

File: Casks/panconvert.rb (hunk 4 above)
Comment (napaxton) [inline Casks/panconvert.rb]: If I do that, why would I still have `version 0.1.5` instead of `version :latest`?

Comment (jawshooah) [inline Casks/panconvert.rb]: It's just string interpolation. The point is to specify the version exactly once, in the `version` stanza, and reuse that value where needed.

File: Casks/panconvert.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/panconvert.rb]: Don’t worry anymore, looks like it is fixed.

**Comment by vitorgalvao** [comment]
Thank you. There are still some audit and formatting issues, but I can take care of those.

**Comment by adidalal** [comment]
prfixmaster fails on these kinds of PRs, last I checked. Errors at applying patch or something. @vitorgalvao

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 22b967fc7c738ff507380a6712b5e701f3403e48. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by vitorgalvao** [comment]
@adidalal Usually, you just need to keep `git am --skip`ping. That’s one of the reasons I’m wary of reducing verbosity in `prfix(master|branch)`, unlike what I’ve done recently with `fastmerge`. It isn’t really `fastmerge`’s fault, but of the commits in the PR and how git handles it (not that it’s wrong, or anything, just providing context).

**Comment by napaxton** [comment]
Why is the checkpoint different?

	I see the singel quotes I left out, so I know for the future…

**Comment by vitorgalvao** [comment]
> Why is the checkpoint different?

Because yours was pointing to a webpage. This is the raw file, which is not only smaller, but less prone to give false positives.

**Comment by napaxton** [comment]
How do I find that checkpoint? For the github webpage enclosgin the raw, it's obvious. (I"m pretty good with Terminal, but less so with git and have been using a GUI.)
> Why is the checkpoint different?
> 
> Because yours was pointing to a webpage. This is the raw file, which is not only smaller, but less prone to give false positives.
>

**Comment by vitorgalvao** [comment]
> How do I find that checkpoint?

[Docs](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Pull request closed without merging**